PublicMapping/districtbuilder
# List all published maps

## Overview

- Adds a new endpoint, `api/globalProjects` with server-side pagination
- Adds a new page in the frontend `/global` containing a display of all projects paginated with a limit of 10 projects per page
- Created a new reusable `PaginationFooter` component 


### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/119730839-328bac00-be44-11eb-890b-08937a322caf.png)
![image](https://user-images.githubusercontent.com/66973361/119730869-39b2ba00-be44-11eb-8826-c58a60bd1434.png)

### Notes

I struggled a bit with the pagination implementation in Redux - initially, I was thinking that `globalProjectsFetchPage` should trigger `globalProjectsFetch`, but ended up having this handled through a param function + `useEffect` in `PaginationFooter`. After further reflection, this actually might not be a bad thing as far as making the component modular for usage in other pagination implementations - but we probably should clean up / rename the actions if we want to stick with this method.


## Testing Instructions

- Go to http://localhost:3003/global
- Make sure that you have at least 11 maps; expect pagination footer to be displayed
- Click on page 2 in the pagination footer; expect maps displayed on screen to change

Closes #719
